baseband/vdif/header.py

```diff
@@ -28,6 +32,41 @@
                   precision=9)
 
 
+_vdif_edv_header_classes = {}
```

Comment:
I realize this is the same as I had, but I would make this a normal property; following precedent in `astropy.time`, maybe just `VDIF_HEADER_CLASSES`.